File: Strings/2591 - HameKameKa/2591.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int C = scanner.nextInt();

        for (int i = 0; i < C; i++) {
            String attack = scanner.next();
            int group1 = attack.indexOf('m') - 1;

            int group2 = attack.substring(attack.indexOf('k')).indexOf('m') - 1;

            StringBuilder result = new StringBuilder();

            result.append('k');
            for (int j = 0; j < group1 * group2; j++) {
                result.append('a');
            }

            System.out.println(result.toString());
        }
        scanner.close();
    }
}

File: Strings/2591 - HameKameKa/2591.cs
using System;

class Program
{
    static void Main()
    {
        int C = int.Parse(Console.ReadLine());
        for (int i = 0; i < C; i++)
        {
            string attack = Console.ReadLine();
            int group1 = attack.IndexOf('m') - 1;

            int group2 = attack.Substring(attack.IndexOf('k')).IndexOf('m') - 1;

            System.Text.StringBuilder result = new System.Text.StringBuilder();

            result.Append('k');
            for (int j = 0; j < group1 * group2; j++)
            {
                result.Append('a');
            }

            Console.WriteLine(result.ToString());
        }
    }
}